lib/migration.pm

```diff
@@ -154,7 +154,8 @@ sub record_disk_info {
     if ($out =~ /btrfs/) {
         assert_script_run 'btrfs filesystem df / | tee /tmp/btrfs-filesystem-df.txt';
         assert_script_run 'btrfs filesystem usage / | tee /tmp/btrfs-filesystem-usage.txt';
-        assert_script_run('snapper list | tee /tmp/snapper-list.txt', 180) unless (is_sles4sap());
+        assert_script_run("(snapper --help | grep -q -- --disable-used-space && snapper list --disable-used-space || snapper list) | tee /tmp/snapper-list.txt", 180) unless (is_sles4sap());
```

Comment:
Need add comments to describe why we use --disable-used-space here.

Reply:
done

Reply:
I don't think you can pass the test if no this option there. I'd suggest you use script_run (but please correct me if I am wrong)